src/utils.py

```diff
@@ -42,7 +42,10 @@ def get_mapping_set(
 ) -> pd.DataFrame:
     """Load up MedGen mapping set (MedGenIDMappings.txt), with some modifications."""
     # Read
-    df = pd.read_csv(inpath, sep='|').rename(columns={'#CUI': 'xref_id'})
+    df = pd.read_csv(inpath, sep='|').rename(columns={
+        '#CUI_or_CN_id': 'xref_id',
+        '#CUI': 'xref_id'  # 2024/05/19: MedGen Renamed to "#CUI_or_CN_id". Leaving this in case they change back.
```

Comment:
@matentzn FYI / question
I left you off of 'reviewers'. I feel like this one is fine for just Trish and I. Do you want me to make that judgement and just leave you off reviewers and not tag you sometimes? Or do you want to at least be tagged on all my PRs?

Reply:
I am looking randomly at PRs, ideally you slack things to me if you need my input. Note that at the moment I am still responsible nominally for the "integration of external content" portion of Mondo, which is about how externally provided content (like UMLS mappings) make it directly into Mondo. I am, however, happy if this is slowly migrating to Trish's plate. Note, that I still feel responsible for this stuff and if you need me, I will provide feedback!